examples/pytorch/pinsage/layers.py

```diff
@@ -36,23 +36,24 @@ def _init_input_modules(g, ntype, textset, hidden_dims):
             module_dict[column] = m
 
     if textset is not None:
-        for column, field in textset.fields.items():
-            if field.vocab.vectors:
-                module_dict[column] = BagOfWordsPretrained(field, hidden_dims)
-            else:
-                module_dict[column] = BagOfWords(field, hidden_dims)
+        for column, field in textset.items():
+            textlist, vocab, pad_var, batch_first = field            
+#            if vocab.vectors:
+#                module_dict[column] = BagOfWordsPretrained(vocab, hidden_dims)
+#            else:
```

Comment:
Let's simply remove L41-43.

Reply:
Sure. Please see my corresp. comment below in model.py.